cmd/sos_upload.go

```diff
@@ -125,7 +129,7 @@ var sosUploadCmd = &cobra.Command{
 		)
 		taskWG.Add(lenFileToUpload)
 
-		workerSem := make(chan int, parallelTask)
+		workerSem := make(chan int, 4)
```

Comment:
having a proper variable for this value helps.

Reply:
yes `4` was for testing